testdata/project-v2/config/webhook/manifests.yaml

```diff
@@ -1,52 +0,0 @@
-
----
```

Comment:
Why this file got dropped?
It should be generated by make manifests.

Reply:
Strange, I probably didn't run `make generate` properly. Re-ran the whole thing and it seems to be back. Thanks for catching it.